src/views/v2/tickets/submit/button.php

```diff
@@ -8,48 +8,47 @@
  *
  * See more documentation about our views templating system.
  *
- * @link http://m.tri.be/1amp
+ * @link    http://m.tri.be/1amp
  *
- * @since TBD
+ * @since   TBD
  *
  * @version TBD
  *
- * @var bool   $must_login If the login is required to purchase tickets.
+ * @var bool $must_login           If the login is required to purchase tickets.
  * @var string $submit_button_name The name of the button.
+ * @var bool $is_modal             True if it's in modal context.
  */
 
-/** @var \Tribe__Tickets__Attendee_Registration__Main $attendee_registration */
-$attendee_registration = tribe( 'tickets.attendee_registration' );
-
 if (
-	! empty( $must_login )
-	|| $attendee_registration->is_modal_enabled()
+		! empty( $must_login )
+		|| $is_modal
 ) {
 	return;
 }
 
 /**
  * Allow filtering of the button classes for the tickets block.
  *
+ * @param array $button_name The button classes.
+ *
  * @since 4.11.3
  *
- * @param array $button_name The button classes.
  */
 $button_classes = apply_filters(
-	'tribe_tickets_ticket_block_submit_classes',
-	[
-		'tribe-common-c-btn',
-		'tribe-common-c-btn--small',
-		'tribe-tickets__buy',
-	]
+		'tribe_tickets_ticket_block_submit_classes',
+		[
```

Comment:
:no_entry_sign: Multi-line function call not indented correctly; expected 4 spaces but found 8

<sub>hash: 78c6f6a5258d68069d6a9c80d84e0b59</sub>